www/artworks/post.php

```diff
@@ -112,19 +42,7 @@ function parseArtworkTags(): array{
 		$_SESSION['artwork'] = $artwork;
 	}
 
-	if (isset($artwork->ArtworkId)){
-		// clean up the uploaded file(s)
-		unlink(WEB_ROOT . $artwork->ImageUrl);
-		unlink(WEB_ROOT . $artwork->ThumbUrl);
-
-		// remove database entry
-		$artwork->Delete();
-	}
-
-	if (isset($artist->ArtistId)){
-		$artist->DeleteIfUnused();
-	}
-
+} finally{
 	http_response_code(303);
-	header('Location: /artworks/new');
+	header('Location: ' . '/artworks/new');
```

Comment:
nit: Let's revert this line. The string concatenation isn't joining a string and a variable.